### ISSUE TITLE
test getting rid of all the n^2 nubs

### ISSUE THREAD

**Issue by kjameslubin** [blockapps/hserver-eth#51]

**Comment by jamshidh** [comment]
Could you fill in the description....

**Comment by kjameslubin** [comment]
I don't think this is worth thinking about at the moment. I may have done it already. The haskell `nub` operation on lists is slow, but I was using it instead of SELECT UNIQUE for some historical reason I can't remember. And again, I might have done it already.
